### DISCUSSION TITLE
Set the random seed for MissForest

### DISCUSSION THREAD

**Discussion by feizheng0209** [HindyDS/MissForest#65] [Q&A]
Hello. I have a question on the reproducibility of the imputation result. It seems the random state for LGBMClassifier/LBGMRegressor are not set in the algorithm, and by default they are set to None, which means that the result will vary between different experiments. 
Is there a proper way to set the random seed for MissForest? Or please tell me if I misunderstand anything.
Thank you.

**Comment by yuenshingyan** [comment]
I think you can change the random seeds using the example from 'Imputing with other estimators'. You just need to replace the estimators with LGBMs.
